tests/python_client/milvus_client/test_milvus_client_collection.py

```diff
@@ -1209,6 +1420,434 @@ def test_milvus_client_load_partially_loaded_collection(self):
+                3. search on the released partition and report error
+                4. load partitions
+        expected: No exception
+        """
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        partition_name_1 = cf.gen_unique_str("partition1")
+        partition_name_2 = cf.gen_unique_str("partition2")
+        self.create_collection(client, collection_name, default_dim)
+        self.create_partition(client, collection_name, partition_name_1)
+        self.create_partition(client, collection_name, partition_name_2)
+        self.load_collection(client, collection_name)
+        self.release_partitions(client, collection_name, [partition_name_1])
+        # Search on released partition should fail
+        vectors_to_search = np.random.default_rng(seed=19530).random((1, default_dim))
+        error = {ct.err_code: 201, ct.err_msg: "partition not loaded"}
+        self.search(client, collection_name, vectors_to_search,
+                   limit=default_limit, partition_names=[partition_name_1],
+                   check_task=CheckTasks.err_res, check_items=error)
+        # Load partitions
+        self.load_partitions(client, collection_name, [partition_name_1, partition_name_2])
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_milvus_client_load_collection_after_release_partition_collection(self):
+        """
+        target: test load collection after release partition and collection
+        method: 1. load collection
+                2. release partition
+                3. query on the released partition and report error
+                4. release collection
+                5. load collection
+        expected: No exception
+        """
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        partition_name_1 = cf.gen_unique_str("partition1")
+        partition_name_2 = cf.gen_unique_str("partition2")
+        self.create_collection(client, collection_name, default_dim)
+        self.create_partition(client, collection_name, partition_name_1)
+        self.create_partition(client, collection_name, partition_name_2)
+        self.load_collection(client, collection_name)
+        self.release_partitions(client, collection_name, [partition_name_1])
+        # Query on released partition should fail
+        error = {ct.err_code: 65538, ct.err_msg: 'partition not loaded'}
+        self.query(client, collection_name, filter=default_search_exp, 
+                   partition_names=[partition_name_1],
+                   check_task=CheckTasks.err_res, check_items=error)
+        self.release_collection(client, collection_name)
+        self.load_collection(client, collection_name)
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_milvus_client_load_partitions_after_release_partition_collection(self):
```

Comment:
can we combine all the release and load partition/collection tests into 1 or 2?  
verify 1.  released partition/collection can not be search/query
          2. loaded partition/collection can be search/query
          3. loaded partition/collection can be loaded again, and released again
          4. released partition/collection can be loaded again, and released again